doc/tutorials/using_cat_on_up2.rst

```diff
@@ -0,0 +1,157 @@
+.. _using_cat_up2:
+
+Using CAT on UP2
+################
+
+The UP Squared board `(UP2) <https://up-board.org/upsquared/specifications/>`_
+is an x86 maker board based on the Intel Apollo Lake platform, which support
+Cache Allocation Technology(CAT). With this feature, the usage of cache can be
+restricted to each VM. ACRN hypervisor can do that by giving VM a cache mask,
+so that the VM will not evict the masked cache lines, as shown in :numref:
+`cache-mask-vm`:
+
+.. figure:: images/using_cat_up2.png
+   :align: center
+   :name: cache-mask-vm
+
+   Cache usage with Cache Mask
+
+
+CAT Support on ACRN
+*******************
+
+As described at ``Intel® 64 and IA-32 Architectures Software Developer’s Manual``,
+chapter 17.17, volume 3. There are 3 steps to use CAT: detecting the CAT capability;
```

Comment:
This would read better as:
```
There are 3 steps to use CAT: 

1. detecting the CAT capability;
2. setup cache mask array MSRs, also know as the Class of Service (CLOS) array; 
3. select one of the CLOS array members that will be the cache mask of the CPU.
```

Reply:
OK,  I will use that